File: samples/CS/07-RibbonColor/Form1.cs
using System;
using System.Drawing;
using System.Windows.Forms;
using WinForms.Ribbon;

namespace _07_RibbonColor
{
    public partial class Form1 : Form
    {
        private RibbonItems _ribbonItems;

        public Form1()
        {
            InitializeComponent();
            _ribbonItems = new RibbonItems(_ribbon);
            _ribbonItems.Init();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            _ribbonItems.Load();
        }
    }
}

File: samples/VB/07-RibbonColor/Form1.vb
Imports WinForms.Ribbon

Namespace _07_RibbonColor
	Partial Public Class Form1
		Inherits Form
		Private _ribbonItems As RibbonItems

		Public Sub New()
			InitializeComponent()
			_ribbonItems = New RibbonItems(_ribbon)
			_ribbonItems.Init()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			_ribbonItems.Load()
		End Sub
	End Class
End Namespace
